docs/source/installation.rst

```diff
@@ -1,13 +1,10 @@
-Getting Started
-===============
+Installation
+======================================
 
 .. note:: 
    If you are completely unfamiliar with the command line or basic git
    usage, see the Tutorial page before proceeding. 
 
-Installation
-------------
-
 We only provide official support for Ubuntu 20.04 due to ROS2. Make sure you
 are on an Ubuntu 20.04 machine before continuing. Alternatively for
 Windows users, using WSL2 with Ubuntu 20.04 will work. The steps to set this
```

Comment:
thoughts on adding a sentence about Macs? "For Mac users, Ubuntu 20.04 can be emulated in a virtual machine; on M1 macs using the arm64 version with the application UTM will work."

Reply:
sure, will add!